flink-java/src/main/java/org/apache/flink/api/java/typeutils/runtime/KryoSerializer.java

```diff
@@ -185,6 +187,8 @@ private void checkKryoInitialized() {
 			this.kryo.setRegistrationRequired(false);
 			this.kryo.register(type);
 			this.kryo.setClassLoader(Thread.currentThread().getContextClassLoader());
+
+			kryo.register(DateTime.class, new JodaDateTimeSerializer());
```

Comment:
I would suggest to add some more serializers from `de.javakaffee` .. since we have it already as a dependency, it doesn't hurt to add them. 
I'm suggesting 
- `jodatime/JodaIntervalSerializer`,
- `guava/ImmutableListSerializer`,
- `UnmodifiableCollectionsSerializer`, 
- `GregorianCalendarSerializer`,
- `EnumSetSerializer`,
- `EnumMapSerializer`,
- BitSetSerializer - serializer for java.util.BitSet
- RegexSerializer - serializer for java.util.regex.Pattern
- URISerializer - serializer for java.net.URI
- UUIDSerializer - serializer for java.util.UUID